packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.base.tsx

```diff
@@ -23,6 +23,7 @@ export class DocumentCardBase extends BaseComponent<IDocumentCardProps, any> imp
 
   public render(): JSX.Element {
     const { onClick, onClickHref, children, type, accentColor, styles, theme, className } = this.props;
+    const nativeProps = getNativeProps(this.props, divProperties, ['onClick', 'type']);
```

Comment:
@dzearing - If I didn't exclude the "type" property the snapshot tests would suddenly start including a type={0} in the resulting snapshot. Do you know if this is a bug with getNativeProps? It seemed unexpected, so Mark and I decided to exclude the property to be on the safe side

Reply:
We figured it out, we didnt notice the "type" property which was already defined in the properties type